### ISSUE TITLE
Setup tool: setting voxel size via command line is ignored & recommission unit tests

### ISSUE THREAD

**Issue by schmie** [schmie/hemeMove#436]
**Reported by miguel on 16 Jul 2013 12:16 UTC**
Unit tests are failing with
```
ImportError: dlopen(/Users/mobernabeu/Documents/workspace/hemelb-dev/hemelb/Tools/setuptool/HemeLbSetupTool/Model/_Generation.so, 2): Symbol not found: __ZN3MPI3Win4FreeEv
```

**Comment by schmie** [comment]
**Modified by miguel on 16 Jul 2013 12:30 UTC**

**Comment by schmie** [comment]
**Comment by miguel on 16 Jul 2013 12:30 UTC**
Bug fixed in d13da359f2b4 and some refactoring. Rupert please CR.

**Comment by schmie** [comment]
**Comment by rupert on 16 Jul 2013 12:42 UTC**
Looks good to me.

THe MPI issues have been fixed in 4f46a532e534

Please check that!

**Comment by schmie** [comment]
**Comment by miguel on 16 Jul 2013 16:59 UTC**
Replying to [rupert](comment:3):
> THe MPI issues have been fixed in 4f46a532e534

Given your refactor, do we still need the #define HEMELB_MPI in `mpiInclude.h`? Happy to go either way.

No setup tool test was exercising the definition of voxel size via command line, cause my code didn't even run! Fixed in 0f0b07e340c6.

Feel free to close.

**Comment by schmie** [comment]
**Modified by rupert on 28 Mar 2014 17:48 UTC**
